include/glow/Backends/BackendOptions.h

```diff
@@ -16,7 +16,19 @@
 #ifndef GLOW_BACKENDS_BACKENDOPTIONS_H
 #define GLOW_BACKENDS_BACKENDOPTIONS_H
 
+#include "llvm/ADT/SmallVector.h"
+
 namespace glow {
+class Storage;
+
+/// Hints provided to the Backend, the backend is not required to honor them.
+struct BackendHints {
+  /// Number of execution units to reserve, 0 for unspecified.
```

Comment:
Can you add some details. What is "execution units to reserve"?
How is it used.

Reply:
I updated the comment, it's a bit vague because each vendor may have a different name or way of subdividing a device. This would be like reserving cores on a CPU.